app/models/test_track/remote/assignment.rb

```diff
@@ -1,14 +1,18 @@
 class TestTrack::Remote::Assignment
   include TestTrack::RemoteModel
 
-  attributes :visitor_id, :split_name, :variant, :context, :unsynced
+  attributes :split_name, :variant, :context, :unsynced
 
-  validates :visitor_id, :split_name, :variant, :mixpanel_result, presence: true
+  validates :split_name, :variant, :mixpanel_result, presence: true
```

Comment:
do these validations ever get invoked?

Reply:
No because we never persist this model. But maybe they used to? Perhaps when we switched to using AssignmentEvent, we stopped persisting these.